src/scripts/mongo/verify.js

```diff
@@ -9,11 +9,11 @@ function verify (email, callback) {
     const users = db.collection('users');
     const query = { email: email, email_verified: false };
 
-    users.update(query, { $set: { email_verified: true } }, function (err, count) {
+    users.update(query, { $set: { email_verified: true } }, function (err, result) {
       client.close();
 
       if (err) return callback(err);
-      callback(null, count > 0);
+      callback(null, result.result.n > 0);
```

Comment:
`result.result.n` is the number of matched documents, so if `email_verified` is already true, then `result.result.n` will be 0. Is this OK?

Reply:
This is the behavior we adopted for the other verify scripts.  I think we can revisit this holistically in the follow-up ticket.  That may require changes to all the scripts potentially.

Reply:
For that follow-up story, we might want to consider using `result.result.nModified` or `result.result.ok`.